src/features/jobs/jobs.py

```diff
@@ -18,24 +18,44 @@
 def start(job_dmss_id: str):
     """Start a job.
 
-    job_dmss_id should be on the format: {data_source_id}/{id}.{attribute}
+    To start the job, a job handler needs to be implemented for the job entity referenced with 'job_dmss_id'.
+    After the job is started, the internal job uid is included in the response. This uid can be used to
+    get status, remove the job or get the result.
+
+    - **job_dmss_id**: an address to a job entity stored in DMSS:
+       - By id: PROTOCOL://DATA SOURCE/$ID.Attribute
+       - By path: PROTOCOL://DATA SOURCE/ROOT PACKAGE/SUB PACKAGE/ENTITY.Attribute
+
+    The PROTOCOL is optional, and the default is dmss.
     """
     return start_job_use_case(job_dmss_id=job_dmss_id).dict()
 
 
 @router.get("/{job_uid}", operation_id="job_status", response_model=StatusJobResponse)
 @create_response(JSONResponse)
 def status(job_uid: str):
+    """Get the status for an existing job.
+
+    - **job_uid**: the job API's internal uid for the job.
+    """
     return status_job_use_case(job_id=job_uid).dict()
 
 
 @router.delete("/{job_uid}", operation_id="remove_job")
 @create_response(PlainTextResponse)
 def remove(job_uid: str):
+    """Remove an existing job by calling the remove() function in the job handler for a given job.
```

Comment:
What does it mean to remove a job? Do you stop it? Remove the data it generated?

Reply:
it will delete the job from the redis database inside the job service class. Will update the docstring!